### DISCUSSION TITLE
Which VS Code browser preview is used in Tailwind CSS: From Zero to Production?

### DISCUSSION THREAD

**Discussion by leifhanack** [tailwindlabs/tailwindcss#3841] [Help]
Hi there, 
I took the course [Tailwind CSS: From Zero to Production ](https://www.youtube.com/watch?v=elgqxmdVms8&list=PL5f_mz_zU5eXWYDXHUDOLBE0scnuJofO0&index=1)
and like to know what browser preview is used. 

![image](https://user-images.githubusercontent.com/128227/112494722-92ea6880-8d83-11eb-803a-9ecb571646f9.png)

The VS Code build-in Simple Browser don't display width and height and so it's difficult to me to know in which screen breakpoint I'm in.

Thanks a lot, 
Leif

**Comment by simonswiss** [comment]
Hey,

That's a good question! To be fair, I'm not quite sure how to answer this one. 

This vscode extension is something @bradlc put together internally very quickly (literally a couple of hours) to support the "Tailwind CSS: From Zero to Production" course. It is heavily inspired by what we have in Tailwind Play.

After a very rapid build, it hasn't really been iterated on since. It would probably need some more work if it was to be released as an extension, and there are no concrete plans to do that in the immediate future, considering other priorities we have.

I don't want to make false promises that we'll release it publicly because we have not properly discussed internally about what we should do with it, beyond using it to produce our videos.

Not sure if this answer helps or confuses you! 🤣

**Reply by leifhanack** [reply]
Thank you Simon. At least I know that I can't find it in the internet 😂  

Maybe we can create an issue (feature request) for it which can be watched. If someday you'll release it good, and if you decide otherwise good as well. 🤷
